src/views/include/sign-out.html

```diff
@@ -3,7 +3,7 @@
     <li class="govuk-list govuk-!-margin-top-2">
       <p class="govuk-body">
         {{ userEmailAuth | default(userEmail) }} | 
-        <a class="govuk-link govuk-link--no-underline govuk-link--no-visited-state" href={{ ROUTES.SIGN_OUT_URL }}>Sign out</a>
+        <a class="govuk-link govuk-link--no-underline govuk-link--no-visited-state" href="https://cola.service.cabinetoffice.gov.uk/v2/github-requests/?logout=true">Sign out</a>
```

Comment:
I would refactor the following environment variable, AUTH_SIGN_IN_URL, as follows:

```js
AUTH_PATH = "https://cola.service.cabinetoffice.gov.uk/v2/github-requests"
AUTH_SIGN_IN_URL = "${AUTH_PATH}/login"
AUTH_SIGN_OUT_URL = "${AUTH_PATH}/?logout=true"
```
and used AUTH_SIGN_OUT_URL on the `href`